Feature/254

jrmartin opened the pull request:
Fixes Docker image for live dev deployment

jrmartin committed:
Remove --quiet flag for test

jrmartin committed:
Attempt to fix dockerfile in live geppetto

jrmartin committed:
Merge remote-tracking branch 'origin/development' into feature/254

jrmartin committed:
Test #2 with livegeppetto docker in development branch

jrmartin committed:
Test #5 - Livegeppetto dockerfile is now the same as the one in geppetto
folder

jrmartin committed:
Test without COPY --chown=1000:1000 GeppettoConfiguration.json
$HOME/workspace/org.geppetto.frontend/src/main/webapp/
 line

jrmartin committed:
Test with contextPath as part of mvn command

jrmartin committed:
Revert last change

jrmartin committed:
Test for Travis: Test installing frontend with 'development' branch.

jrmartin committed:
Test #5

jrmartin committed:
Test without setting context path while running mvn install on frontend

jrmartin committed:
reset maven install for frontend

filippomc reviewed (commented):
@jrmartin can you please help me understand what problem is solving and why the copy and --quiet are fixing it?

jrmartin committed:
Add quiet argument to mvn install

jrmartin commented:
@filippomc The --quiet flag is not fixing anything, I removed it to have more detailed on the logs about the failure. I added it back to keep it consistent with what it was.

The rename.sh -> copy.sh is the fix. Copy.sh downloads the geppetto-client repo into the webapp folder, rename.sh "change geppetto-client target branch on geppetto-application package.json".
